server/src/main/java/com/cloud/resource/ResourceManagerImpl.java

```diff
@@ -2190,6 +2191,12 @@ protected HostVO createHostVO(final StartupCommand[] cmds, final ServerResource
                     hostTags = implicitHostTags;
                 }
             }
+
+            // Update host memory reported by agent
+            if (ssCmd.getHypervisorType().equals(HypervisorType.KVM) ||
+                    ssCmd.getHypervisorType().equals(HypervisorType.LXC)) {
+                host.setDom0MinMemory(Long.parseLong(_configDao.getValue(HOST_RESERVED_MEM_MB_STRING)) * 1024L * 1024L);
```

Comment:
We can use `ByteScaleUtils` for scale conversions.

Reply:
Done 👍